tests/jobs/test_scheduler_job.py

```diff
@@ -1718,7 +1718,6 @@ def test_execute_task_instances_limit(self):
             ti.refresh_from_db()
             self.assertEqual(State.QUEUED, ti.state)
 
-    @pytest.mark.quarantined
```

Comment:
Should I remove this change from here?

Reply:
Probably not as part of this PR -- we don't know if its stable yet.

Reply:
Updated: https://github.com/astronomer/airflow/pull/956/commits/6dd35bcc3986429ebed15b7287fc204a7d4fdcb0